test/setup.js

```diff
@@ -1,5 +1,3 @@
-// Required since lp-hoc > lp-requests uses async/await
-import 'core-js/stable'
```

Comment:
We removed the `lp-hoc` dependency awhile back when we brought the cloudinary HOC into this package. I removed this and ran tests successfully on this branch and also removed it on main and ran tests successfully